### ISSUE TITLE
MacOS X Mojave support

### ISSUE THREAD

**Issue by krtek** [adam-p/markdown-here#529]
Safari in MacOS X Mojave no longer supports side-loaded extensions. The only way how to install an extension is only via App Store. Could you release Markdown Here through App Store?

As a workaround, one can pose as extension developer, unzip the extension and [load it via Extension Builder](https://georgegarside.com/blog/macos/install-any-safari-extension-macos-mojave/).

**Comment by juanmirocks** [comment]
Yes please 🙏 I miss my awesomely-formatted emails in Mojave Safari

How can I help with the task?

**Comment by Sweeper777** [comment]
+1

**Comment by jchekanoff** [comment]
+1

**Comment by bbakernoaa** [comment]
+1

**Comment by orrp** [comment]
+1

**Comment by rogerluan** [comment]
Hey everyone 👋 

I ended up implementing a proper Safari Extension for markdown formatting and released it in the official Mac App Store:

https://apps.apple.com/us/app/markdown-email/id6741367002

<img width="1582" alt="Image" src="https://github.com/user-attachments/assets/5431c3f3-78a4-428a-aaae-cae1bb4e337d" />

Since being an Apple Developer has significant recurring cost, I put it for sale for the price of a lunch 🙏

Also, since it's now a first-class citizen of the Apple ecosystem (being distributed by the official Mac App Store), you won't get any errors/warnings like the one mentioned here, aside from the regular permission request from Safari that every extension needs 😊
